rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
```

Comment:
I don't think we would want to have nightly toolchain by default.  Instead, I think it would be better to add a `justfile` recipe to compile `simd` version -- and pass all the needed parameters via the command line.